use-cases/client-server-msgs-perf/svmsg_file_server.c

```diff
@@ -136,18 +154,33 @@ struct option Long_options[] = {
     , { "clock-thread-cputime-id"   , no_argument         , NULL, 't'}
 
     // Options that need arguments
+    , { "num-server-threads"        , required_argument   , NULL, 'n'}
     , { "perf-outfile"              , required_argument   , NULL, 'o'}
     , { NULL, 0, NULL, 0}           // End of options
 };
 
-const char * Options_str = "o:dhmprt";
+const char * Options_str = "no:dhmprt";
 
 // Useful macros
 #define ARRAY_LEN(arr)  (sizeof(arr) / sizeof(*arr))
 
+#if defined(L3_LOGT_SPDLOG)
+std::shared_ptr<spdlog::logger> Spd_logger; // = spdlog::basic_logger_mt();
```

Comment:
Made it global, so that the code in the work-horse function implementing the operation, i.e., `svr_op_incr()` can still reference this spdlog-logging handle when running tests in that build-mode.